docs/list.md

```diff
@@ -94,6 +94,19 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: List of [hacktoberfest-labeled issues](https://github.com/flyteorg/flyte/issues/2917)
 - **Notes**: Check out the [blog post](https://blog.flyte.org/meet-flyte-and-unionml-at-hacktoberfest-2022) for more info.
 
+### H
+
+#### **Hasura**
+
+
+- **Swag**: Tshirt, Stickers
+
+- **Requirements**: Submit 1 or more pull requests and get them approved/merged to receive a Tshirt.
+- **How to sign up**: visit [this repo](https://github.com/hasura/graphql-engine)     
+- **Issues**: Any issue [listed here](https://github.com/hasura/graphql-engine/issues)
```

Comment:
"Any issue listed here...." . Please write in complete sentences with punctuation. 👍